app.js

```diff
@@ -2,14 +2,14 @@
 
```

Comment:
This would take a refactor in future versions but the way the variables are defined here is a bit too verbose, if it was all in one object it could be easier to use and maintain, i.e the standard "M" in the MVC architecture.